mipengine/controller/api/algorithm_request_dto.py

```diff
@@ -1,28 +1,20 @@
-from dataclasses import dataclass
-from dataclasses import field
+from pydantic import BaseModel
 from typing import Any
 from typing import Dict
 from typing import List
-from typing import Optional
 
-from dataclasses_json import dataclass_json
 
-
-@dataclass_json
-@dataclass
-class AlgorithmInputDataDTO:
+class AlgorithmInputDataDTO(BaseModel):
     pathology: str
     datasets: List[str]
-    filters: Optional[Any] = None
-    x: Optional[List[str]] = field(default_factory=list)
-    y: Optional[List[str]] = field(default_factory=list)
+    filters: dict = None
+    x: List[str]
```

Comment:
@jassak is there a chance that x could also be null? Should we make that optional as well?